CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 0.19.0 (July 6, 2022)
+
 ### Breaking Changes
 
 - [\514](https://github.com/cosmos/iavl/pull/514) Downgrade Tendermint to 0.34.x
```

Comment:
```suggestion
- [\514](https://github.com/cosmos/iavl/pull/514) Downgrade Tendermint to 0.34.x
- [\500](https://github.com/cosmos/iavl/pull/500) Return errors instead of panicking. 
```